src/main/resources/react4xp/_entries/Dashboard/utils/websocket/WebsocketProvider.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { createContext } from 'react'
+import PropTypes from 'prop-types'
+import { actions as commonActions } from '../../containers/HomePage/slice'
+import setupStatRegListeners from '../../containers/StatRegDashboard/listeners'
+import setupHomePageListeners from '../../containers/HomePage/listeners'
+import setupDataQuieriesListeners from '../../containers/DataQueries/listeners'
```

Comment:
setupDataQuieresListeners -> setupDataQueriesListeners?  @ both import and function call